src/Dialect/ONNX/ONNXOps.cpp

```diff
@@ -2687,6 +2687,49 @@ LogicalResult ONNXConstantOp::inferShapes(
 // Concat
 //===----------------------------------------------------------------------===//
 
+static LogicalResult verify(ONNXConcatOp op){
```

Comment:
clang-format will reject this code unfort.
We use clang-format to format the code for us, you can either invoke the command on files you changed from the shell. OR add integration to the editor you are using. I use vscode and there are packages to support clang-format